sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExec.scala

```diff
@@ -75,7 +75,7 @@ object GpuMetric extends Logging {
   val DESCRIPTION_PARTITION_SIZE = "partition data size"
   val DESCRIPTION_NUM_PARTITIONS = "partitions"
   val DESCRIPTION_TOTAL_TIME = "total time"
-  val DESCRIPTION_GPU_OP_TIME = "GPU time"
+  val DESCRIPTION_OP_TIME = "GPU time"
```

Comment:
Are we still going to call it "GPU time"? This is what the user sees, "opTime" is just a key for the map.

Reply:
oops, missed that. updated.